apps/frontend/src/libs/components/avatar/avatar.tsx

```diff
@@ -0,0 +1,28 @@
+import styles from "./styles.module.css";
+
+type Properties = {
+	name?: string;
+};
+
+const FIRST_LETTER_INDEX = 0;
+
+const Avatar = ({ name }: Properties): JSX.Element => {
+	let firstLetter = null;
+
+	if (name && name.length > FIRST_LETTER_INDEX) {
+		firstLetter = name[FIRST_LETTER_INDEX];
+	}
+
+	const hasFirstLetter = Boolean(firstLetter);
```

Comment:
Sign up validation will require us to have at least one letter, so we don't need this check here